properties.schema

```diff
@@ -217,18 +217,42 @@
                 "type": "string",
                 "required": false,
                 "default": "",
-                "title": "Classes",
+                "title": "Graphic Classes",
                 "inputType": "Text",
                 "validators": [],
                 "help": "Allows you to specify custom CSS classes to be applied to the popup item"
               }
             }
           },
+          "_pin": {
+            "type": "object",
+            "required": false,
+            "title": "Item Pin",
+            "properties": {
+              "src": {
+                "type": "string",
+                "required": false,
+                "default": "",
+                "inputType": "Asset:image",
+                "validators": [] ,
+                "help": "The pin image leave blank for default"
```

Comment:
Minor: I think there should be some punctuation in this help text to have it read better

Reply:
:+1: